frontend/src/views/Reporting/Reporting.tsx

```diff
@@ -18,6 +20,28 @@ export default function FieldList() {
   const { state } = useAppState();
   const navigate = useNavigate();
 
+  const unassignedManures = useMemo(() => {
+    const generatedManures = state.nmpFile?.years[0].GeneratedManures || [];
+    const importedManures = state.nmpFile?.years[0].ImportedManures || [];
+    const unassignedM: ManureInSystem[] = [];
+    const assignedM: ManureInSystem[] = [];
+    (generatedManures || []).forEach((manure) => {
```

Comment:
You don't need to do `(generatedManures || [])` here if you're doing `|| []` above.